src/test/java/tools/jackson/databind/struct/UnwrappedEmptyAsNull1709Test.java

```diff
@@ -0,0 +1,122 @@
+package tools.jackson.databind.deser;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import org.junit.jupiter.api.Test;
+import tools.jackson.databind.DeserializationFeature;
+import tools.jackson.databind.ObjectMapper;
+import tools.jackson.databind.testutil.DatabindTestUtil;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+// https://github.com/FasterXML/jackson-databind/issues/1709
+public class UnwrappedEmptyAsNull1709Test extends DatabindTestUtil
+{
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+    static class Container {
+        public String name;
+        @JsonUnwrapped
+        public Unwrapped u;
+    }
+
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+    static class Unwrapped {
+        public String s;
+        public Integer n;
+    }
```

Comment:
Names could be a bit more specific, to help navigating for others easier.

```suggestion
    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
    static class Container1709 {
        public String name;
        @JsonUnwrapped
        public Unwrapped u;
    }

    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
    static class Unwrapped1709 {
        public String s;
        public Integer n;
    }
```

Reply:
That makes sense.
I will make sure to use more specific names and include the issue number in class names going forward :)